src/config.rs

```diff
@@ -166,6 +167,29 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
```

Comment:
Should I have it for `program` as well? It doesn't seem to need it from some quick testing..